apps/src/aichat/views/modelCustomization/SaveChangesAlerts.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {useSelector} from 'react-redux';
+
+import {
+  selectCurrentEmptyModelCard,
+  selectSavedEmptyModelCard,
+  selectHavePropertiesChanged,
+} from '@cdo/apps/aichat/redux/aichatRedux';
+import Alert from '@cdo/apps/componentLibrary/alert/Alert';
+import {isReadOnlyWorkspace} from '@cdo/apps/lab2/lab2Redux';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import {isDisabled} from './utils';
+
+import styles from '../model-customization-workspace.module.scss';
+
+const SaveChangesAlerts: React.FunctionComponent = () => {
+  const visibility = useAppSelector(
+    state => state.aichat.fieldVisibilities.modelCardInfo
+  );
+  const isReadOnly = useSelector(isReadOnlyWorkspace) || isDisabled(visibility);
+  const saveInProgress = useAppSelector(state => state.aichat.saveInProgress);
+  // const currentSaveType = useAppSelector(state => state.aichat.currentSaveType);
+  const havePropertiesChanged = useAppSelector(selectHavePropertiesChanged);
+  const isCurrentModelCardEmpty = useAppSelector(selectCurrentEmptyModelCard);
+  const isSavedModelCardEmpty = useAppSelector(selectSavedEmptyModelCard);
+  return (
```

Comment:
nit: thoughts on computing alert `text` and `info` separately and then returning a single `Alert`? i.e. something like
```
  const unsaved = havePropertiesChanged && !saveInProgress && !isReadOnly;
  const empty =
    isCurrentModelCardEmpty &&
    isSavedModelCardEmpty &&
    !saveInProgress &&
    !isReadOnly;
  const saved =
    !isSavedModelCardEmpty &&
    !havePropertiesChanged &&
    !saveInProgress &&
    !isReadOnly;

  const [text, type]: [string, AlertProps['type']] | [] = unsaved
    ? ['You have unsaved changes', 'warning']
    : empty
    ? ['Remember to save your changes', 'info']
    : saved
    ? ['Saved', 'success']
    : [];

  if (text && type) {
    return (
      <Alert text={text} type={type} size="s" className={styles.saveAlert} />
    );
  }

  return null;
```